lib/eventq/eventq_aws/aws_queue_worker.rb

```diff
@@ -96,27 +96,27 @@ def process_message(msg, poller, queue, block)
           when :accepted
             # Acceptance was handled directly when QueueWorker#process_message was called
           when :reject
-            reject_message(queue, poller, msg, retry_attempts, message, message_args.abort)
```

Comment:
Should the call `.abort` have been removed?

Reply:
yeah it was not actually being used within the `reject_message` method (`.abort` is an attribute not a method). If required it can now be set via `args.abort= true` etc but crucially we can now access the message id from`args.id`